haskell_102/codelab/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM haskell:8
+
+RUN cabal update && cabal install random
+WORKDIR /codelab
+
+COPY . .
```

Comment:
Isn't this a no-op? (I'm no Docker expert, so maybe I'm missing something)

Reply:
`COPY <src> <dest>`
This copies the entire directory from the codelab folder of this repo in to the current folder of the container, which in this case is `/codelab`
See COPY for more https://docs.docker.com/engine/reference/builder/#copy

Reply:
Can you add a short comment for this, please? To not trip future readers.

Thank you for the clarification

Reply:
Sure thing 👍